torch/_inductor/compile_fx.py

```diff
@@ -1269,7 +1269,7 @@ def compile_fx(
                     )
                     == "1",
                     "triton.autotune_cublasLt": False,
-                    "triton.cudagraphs": False,
+                    "triton.cudagraphs": True,
```

Comment:
I dont think we want to patch this to True or False, just leave it to what was previously configured.

Reply:
Sorry, this is another that I was planning on removing with the WIP, was just there to test in the perf dashboard. I'll add a `# TODO remove before merging` next time.